examples/Makefile-cc2538

```diff
@@ -67,6 +67,10 @@ COMMONCFLAGS                   := \
     -I$(CONFIG_FILE_PATH)         \
     $(NULL)
 
+ifeq ($(CERT_LOG),1)
+COMMONCFLAGS += -DOPENTHREAD_CONFIG_LOG_OUTPUT=OPENTHREAD_CONFIG_LOG_OUTPUT_APP
```

Comment:
Should this be in [examples/common-switches.mk](https://github.com/openthread/openthread/blob/master/examples/common-switches.mk)?